setup.py

```diff
@@ -38,13 +38,10 @@ def get_version():
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
         'Natural Language :: English',
-        "Programming Language :: Python :: 2",
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Again, I think this should have been updated in the master branch already - can you double check by merging that in here?

Reply:
Not in master yet:
https://github.com/FAST-HEP/fast-carpenter/blob/master/setup.py#L47